package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "kitty-items",
   "version": "0.1.0",
-  "dependencies": {},
+  "dependencies": {
+    "yargs": "^16.2.0"
+  },
   "devDependencies": {
-    "concurrently": "^6.0.0"
+    "@types/node": "^14.14.37",
+    "@types/yargs": "^16.0.1"
   },
   "scripts": {
-    "start:api": "cd ./api && npm run start:dev",
-    "start:web": "cd ./web && npm run start:dev",
-    "start:wallet": "cd ./fcl-dev-wallet && npm run start:dev",
-    "start": "concurrently \"npm:start:*\""
+    "build": "./heroku-build.sh",
```

Comment:
It feels like a bit of a circular dependency to have `npm` call a bash script that calls `npm`. Is it done this way because Heroku calls `npm run build` by default?

Users may run this command outside of a Heroku environment, so just want to make sure it's not overfitting to Heroku requirements.

Reply:
Looks like you can create a customized build process just for Heroku with `heroku-postbuild`: https://devcenter.heroku.com/articles/nodejs-support#customizing-the-build-process

```js
"scripts": {
  "start": "node index.js",
  "build": "ng build",
  "heroku-postbuild": "ng build --prod" // this will be run on Heroku
}
```